docs/proposals/vertical-pod-autoscaler.md

```diff
@@ -0,0 +1,75 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/proposals/vertical-pod-autoscaler.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## Abstract
+
+Vertical Pod Autoscaler is a feature that periodically adjust pods' [Resources](resource-qos.md#resource-specifications) based on demand signals.
+This is a natural extension of [Initial Resources](initial-resources.md) plugin.
+
+**Note:** since there is no plan to implement the feature immediately (as for 9/9/15) this document is just an overview how it should be done
+rather than detailed design.
+
+## Motivation
+
+Having Resources set to a correct value is very important: if the value is too high resources in the cluster are wasted,
+if the value is too low higher latency of the service or even termination due to OOM is observed. Initial Resources plugin
+set Resources but it can be wrong, especially in case when there is not enough data for the proper prediction, so
+there is need to add possibility to adjust the decision later.
+
+## Overview
+
+Vertical Pod Autoscaler will consist of two parts: the common logic which decides when the prediction should be made and actuates the decision
+and Prediction API which will provide resource predictions for pods.
+
+### Common logic
```

Comment:
Why is this called "common logic"? I would just call it "prediction actuation" (determining when to actuate is part of actuating...)